_includes/events.html

```diff
@@ -1,21 +1,15 @@
 {% unless page.permalink == "/events" %}
-  <a href="/events">{% t home.events.more %}</a>
+<a href="/events">{% t home.events.more %}</a>
 {% endunless %}
 <ul
   class="list -no-list-style l-stack -vertical"
   style="--stack-spacing: 1.5rem"
   role="list"
 >
-  {% assign events = site.events | filter_tags: include.tags | reverse %}
-  {% assign future_events = events | where_exp: "event", "event.date >= site.time" %}   
-  
-  {% if include.limit and future_events.size > 0 %}
-    {% for event in future_events reversed limit: include.limit  %}  
-      {% include event-card.html %}
-    {% endfor %} 
-  {% else %}  
-    {% for event in events limit: include.limit  %}
-      {% include event-card.html %}
-    {% endfor %}
-  {% endif %}  
+  {% assign events = site.events | filter_tags: include.tags | reverse %} {%
+  assign future_events = events | where_exp: "event", "event.date >= site.time"
+  %} {% if include.limit and future_events.size > 0 %} {% for event in
+  future_events reversed limit: include.limit %} {% include event-card.html %}
+  {% endfor %} {% else %} {% for event in events limit: include.limit %} {%
+  include event-card.html %} {% endfor %} {% endif %}
```

Comment:
This looks a little weird now. I'm wondering whether prettier can handle these erb templates, or whether we can refactor this. I think we still merge for now and can come back to it later.

Reply:
I needed to add an override so the prettier liquid plugin could apply to .html extensions